src/app/modules/form/admin-form/admin-form.errors.ts

```diff
@@ -29,3 +29,11 @@ export class InvalidCollaboratorError extends ApplicationError {
     super(message)
   }
 }
+
+export class PaymentChannelNotFoundError extends ApplicationError {
+  constructor(
+    message = 'Payment channel not found. Please connect your Payment account in Settings',
```

Comment:
As @staceytan1998 suggested:
```suggestion
    message = 'Please ensure that you have connected your Stripe account in settings to save this field.',
```
Shouldn't confuse users with the foreign concepts like payment channels bah

Reply:
[6aa8f60](https://github.com/opengovsg/FormSG/pull/6258/commits/6aa8f6026f7873f170a2b9c0f2a9fbb6adaccef9)